filestream/filestream_test.go

```diff
@@ -0,0 +1,181 @@
+package filestream
+
+import (
+	"context"
+	"io"
+	"os"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestFileStream_MoreBytes(t *testing.T) {
+	var wg sync.WaitGroup
+	ctx := context.Background()
+	buffer := make([]byte, 100)
+
+	Convey("filestream.MoreBytes()", t, func() {
+		file_name := "/tmp/tail_test.log"
```

Comment:
you can use `t.TempDir()` to create a portable temp dir which will automatically be cleaned up when your test exits